src/index.ts

```diff
@@ -8,12 +10,17 @@ const getUrlString = (query: { [k: string]: string }) => {
 
 export const getRequestUrl = (
   settings: ComponentSettings,
-  payload: MCEvent['payload']
+  payload: MCEvent['payload'],
+  client: Client
 ) => {
   const { advertiserId } = settings
   const { timestamp, groupTag, activityTag, ...customFields } = payload
   const baseURL = 'https://ad.doubleclick.net/activity;'
 
+  if (client.get('_gcl_dc')) {
+    customFields.gcldc = client.get('_gcl_dc')?.split('.').pop()
```

Comment:
I think it would be better to take this a few lines down and write into `query` instead of `customFields`, as it could be misleading in the future. It's also better to call `client.get('_gcl_dc')` just once.

Alternatively write directly when setting up the `query` constant, like this:
```
const gcldc = client.get('_gcl_dc')

const query = {
  ...(gcldc && { gcldc }),
}
```